server/api/support.py

```diff
@@ -133,6 +141,68 @@ def list_organizations():
     )
 
 
+@api.route("/support/organizations/users", methods=["GET"])
+@restrict_access_support
+def list_users_by_organization():
+    string_io = io.StringIO()
+    csv_writer = csv.writer(string_io)
+    headers = ["Organization Name", "Audit Name", "Role", "Email", "Jurisdiction Name"]
+    csv_writer.writerow(headers)
+
+    elections_with_users = (
+        Election.query.filter(
+            Election.deleted_at.is_(None),
+            Election.created_at > datetime.now(timezone.utc) - timedelta(weeks=52),
+        )
+        .options(
+            subqueryload(Election.jurisdictions)
+            # pylint: disable=no-member
+            .subqueryload(Jurisdiction.jurisdiction_administrations).subqueryload(
+                JurisdictionAdministration.user
+            ),
+            subqueryload(Election.organization)
+            # pylint: disable=no-member
+            .subqueryload(Organization.audit_administrations).subqueryload(
+                AuditAdministration.user
+            ),
+        )
```

Comment:
This query is fairly unoptimized but runs locally in < 250ms for all audits in the last year.